lambda_src/lambda_function.py

```diff
@@ -44,7 +44,7 @@ def async_flow_init(event: Any, context: Any) -> Dict[Text, Any]:
         f'geff.drivers.destination_{urlparse(destination).scheme}'
     )
     # Ignoring style due to dynamic import
-    destination_driver.initialize(destination, batch_id)  # type: ignore
```

Comment:
this is a generic API so just because it's vestigial here doesn't mean it won't be useful in another destination driver

Reply:
I'd considered only the current s3 destination driver, will add it back for the future drivers.

Reply:
this seems outside the scope of this PR and could be troublesome to collaborators. not sure if that's the case here, but there might be other drivers in forks or branches of this code and removing this line would prevent them from applying this change to their versions of the code.